config.json

```diff
@@ -130,12 +135,15 @@
   { "target": "libp2p/go-ws-transport" },
   { "target": "libp2p/go-yamux" },
   { "target": "libp2p/hydra-booster" },
+  { "target": "multiformats/go-base32" },
+  { "target": "multiformats/go-base36" },
   { "target": "multiformats/go-multiaddr" },
   { "target": "multiformats/go-multiaddr-dns" },
   { "target": "multiformats/go-multiaddr-fmt" },
   { "target": "multiformats/go-multibase" },
   { "target": "multiformats/go-multicodec" },
   { "target": "multiformats/go-multihash" },
   { "target": "multiformats/go-multistream" },
-  { "target": "multiformats/go-varint" }
+  { "target": "multiformats/go-varint" },
+  { "target": "multiformats/ma-pipe" },
```

Comment:
hmm, trailing newline - should we make CI on PRs fail if the json syntax is invalid?

Reply:
https://github.com/protocol/.github/issues/128 would probably have caught this, as it needs to parse the json anyway.

Reply:
I'm surprised `jq` didn't catch it. It's run as part of check_config.sh.

Reply:
It's especially weird since locally jq fails: `parse error: Expected another array element at line 149, column 1`.

Reply:
Actually, it did throw an error, but for some reason not fail the build: 
![image](https://user-images.githubusercontent.com/1478487/127020757-df1a10ab-e7b2-414b-83a8-7bc187319c27.png)

Reply:
Rebased on top of #149, and CI is now actually failing.